efel/cppcore/LibV1.cpp

```diff
@@ -1470,14 +1465,14 @@ static int __AP_width(const vector<double>& t, const vector<double>& v,
     v.begin() + indices[i + 1], bind2nd(less_equal<double>(), v_hm)));
     apwidth.push_back(t[hm_index2] - t[hm_index1]);
     */
-    int onset_index = distance(
-        v.begin(), find_if(v.begin() + indices[i], v.begin() + indices[i + 1],
-                           bind2nd(greater_equal<double>(), threshold)));
+    auto onset_index = std::distance(
+      v.begin(), std::find_if(v.begin() + indices[i], v.begin() + indices[i + 1],
```

Comment:
why use std:: here? You did not use it for the other changes

Reply:
Done in the next commit. The usage is now consistent within LibV1